### ISSUE TITLE
"New instance" doesn't create anything

### ISSUE THREAD

**Issue by Doodlinka** [zetaPRIME/MultiBound2#14]
I click "new instance", paste a link to my unlisted collection, click "ok", then it waits several seconds, saying 'Updating mods 0/0", and then nothing happens. Nnthing in the UI, nothing in the instances folder.

**Comment by banoargeek** [comment]
Yeah, According to Zeta, Steam changed something in their backend so the program doesn't currently work to fetch mods.

Also, It wouldn't be able to access an unlisted collection. If I recall correctly, it only works for public collections, as you aren't required to log in with the app.

#13
